.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - run: sudo docker run --name mongo -d -p 27017:27017 mongo
+      - name: Checkout Habitica
+        uses: actions/checkout@v2
+        with:
+          path: habitica
+          repository: HabitRPG/habitica
+          ref: 'develop'
+      - run: |
+          cd habitica
+          cp config.json.example config.json
+          npm ci
+          node ./website/server/index.js & cd ..
```

Comment:
`npm start` might be better here ?

Reply:
`npm start` runs `gulp nodemon` . It couldn't get it running on github pipeline.

```
> gulp nodemon

[05:16:33] Using gulpfile ~/work/habitica-zapier/habitica-zapier/habitica/gulpfile.js
[05:16:33] Starting 'nodemon'...
[05:16:33] Finished 'nodemon' after 30 ms
[05:16:33] [nodemon] 1.19.4
[05:16:33] [nodemon] to restart at any time, enter `rs`
[05:16:33] [nodemon] watching dir(s): *.*
[05:16:33] [nodemon] watching extensions: js,mjs,json
[05:16:33] [nodemon] starting `node ./website/server/index.js`
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! habitica@4.136.0 start: `gulp nodemon`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the habitica@4.136.0 start script.
```

Reply:
makes sense, probably `nodemon` doesn't run on GH actions